listenbrainz/db/stats.py

```diff
@@ -24,7 +24,10 @@
 import json
 from typing import Optional
 
+import psycopg2
 import sqlalchemy
+import ujson
+from psycopg2.extras import execute_values
```

Comment:
I guess more changes are coming here?

Reply:
Earlier I put some stuff here but now I think it makes sense to just create a separate file for Year in Music stuff. So no changes to stats.py now.